api/envoy/api/v2/auth/cert.proto

```diff
@@ -172,8 +172,24 @@ message TlsSessionTicketKeys {
   repeated core.DataSource keys = 1 [(validate.rules).repeated = {min_items: 1}];
 }
 
-// [#next-free-field: 9]
+// [#next-free-field: 10]
 message CertificateValidationContext {
+  // Peer certificate verification mode.
+  enum TrustChainVerification {
+    // Perform default certificate verification (e.g., against CA / verification lists)
+    VERIFY_TRUST_CHAIN = 0;
+
+    // Connections where the certificate fails verification will be permitted.
+    // For HTTP connections, the certificate details will be unconditionally placed in the
+    // *x-forwarded-untrusted-client-cert* HTTP header.
+    ACCEPT_UNTRUSTED = 1;
+
+    // Connections will be permitted without certificate verification performed.
```

Comment:
What value does this additional option provide given the semantics above?

Reply:
Removed this option